examples/glamorous-tailwind/src/index.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+import { AppContainer } from 'react-hot-loader'
+
+// Your top level component
+import App from './App'
+
+// Export your top level component (for static rendering)
+export default App
+
+// Render your app
+if (typeof document !== 'undefined') {
+  const render = Comp => {
+    ReactDOM.hydrate(
+      <AppContainer>
+        <Comp />
+      </AppContainer>,
+      document.getElementById('root'),
+    )
+  }
```

Comment:
Awesome stuff!
Could you change this to [reflect the other examples](https://github.com/nozzle/react-static/blob/master/examples/basic/src/index.js#L13-L21)?

Reply:
ok, done. Sorry, I missed that update.